workspace-hack/Cargo.toml

```diff
@@ -250,32 +215,26 @@ serde_test = { version = "1", default-features = false }
 serde_urlencoded = { version = "0.7", default-features = false }
 serde_with = { version = "2", features = ["alloc", "macros", "std"] }
 serde_yaml = { version = "0.8", default-features = false }
-sha2-93f6ce9d446188ac = { package = "sha2", version = "0.10", default-features = false }
-sha2-274715c4dabd11b0 = { package = "sha2", version = "0.9", default-features = false, features = ["std"] }
-sha3-93f6ce9d446188ac = { package = "sha3", version = "0.10", features = ["std"] }
-sha3-274715c4dabd11b0 = { package = "sha3", version = "0.9", default-features = false }
+sha2 = { version = "0.9", default-features = false, features = ["std"] }
+sha3 = { version = "0.9", default-features = false }
 sharded-slab = { version = "0.1", default-features = false }
-signature = { version = "1", features = ["digest", "digest-preview", "rand-preview", "rand_core", "std"] }
+signature = { version = "1", features = ["rand-preview"] }
```

Comment:
maybe @bmwill know - im quite confused why `cargo hakari generate` will generate this line without `features = ["rand-preview"]` and there is a compile error in fastcrypto repo:  

```
error[E0432]: unresolved import `signature::rand_core`
  --> /Users/joy/.cargo/git/checkouts/fastcrypto-52df1bab54ce57e6/0d0d120/src/ed25519.rs:14:17
   |
14 | use signature::{rand_core::OsRng, Signature, Signer, Verifier};
   |                 ^^^^^^^^^ could not find `rand_core` in `signature`
```

when i manually add back the rand-preview feature, it compiles fine. is it because fastcrypto does not have workspace-hack? or how can we make workspace-hack to generate with the feature that its dependency needs?

Reply:
cc @bmwill

Reply:
resolved by explicitly specifying the feature in fastcrypto